pkg/kubectl/describe/versioned/describe.go

```diff
@@ -2949,9 +2962,14 @@ func describeNode(node *corev1.Node, nodeNonTerminatedPodsList *corev1.PodList,
 		w.Write(LEVEL_0, " Kubelet Version:\t%s\n", node.Status.NodeInfo.KubeletVersion)
 		w.Write(LEVEL_0, " Kube-Proxy Version:\t%s\n", node.Status.NodeInfo.KubeProxyVersion)
 
+		// remove when .PodCIDR is depreciated
 		if len(node.Spec.PodCIDR) > 0 {
 			w.Write(LEVEL_0, "PodCIDR:\t%s\n", node.Spec.PodCIDR)
 		}
+
+		if len(node.Spec.PodCIDRs) > 0 {
+			w.Write(LEVEL_0, "PodCIDRs:\t%s\n", strings.Join(node.Spec.PodCIDRs, ","))
```

Comment:
Why is this so casual and pod IPs is more structured?

Reply:
PodCIDRs is just a string slice. PodIPs is a struct and is more ceremonial (also had properties before we remove them).